iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/HubTierConnectionAndroidRunner.java

```diff
@@ -51,16 +44,4 @@ public static Collection inputs() throws IOException, IotHubException, GeneralSe
         identities = getIdentities(inputs);
         return inputs;
     }
-
-    @AfterClass
-    public static void cleanUpResources()
-    {
-        tearDown(identities);
```

Comment:
I assume this cleanup is happening elsewhere now?

Reply:
Actually, this looks like it shouldn't have been deleted. I'll revert this